### PR TITLE
ncm-systemd: add support for AmbientCapabilities and CapabilityBoundingSet

### LINKED ISSUE #1211: ncm-systemd 17.7: AmbientCapabilities service attribute not supported
`AmbientCapabilities` is a service attribute that allows to define a *capability* for a service, for example `CAP_NET_BIND_SERVICE` to allow a service that doesn't run as root to bind a privileged port (<1024). Unformatunately `ncm-systemd` doesn't support it.

### PR DESCRIPTION
**Pull request by jouvin**
Fixes #1211

### CODE REVIEW

**Comment by jouvin** [comment]
Why is `ncm-accounts` tests failing?

**Comment by stdweird** [comment]
all unittests are broken, see https://github.com/quattor/maven-tools/pull/167 for some discussion

File: ncm-systemd/src/main/pan/components/systemd/schema.pan [hunk 1]
```diff
@@ -198,8 +198,10 @@ http://www.freedesktop.org/software/systemd/man/systemd.service.html
 type ${project.artifactId}_unitfile_config_service = {
     include ${project.artifactId}_unitfile_config_systemd_exec
     include ${project.artifactId}_unitfile_config_systemd_kill
+    'AmbientCapabilities' ? string[]
```
Comment (jrha) [inline ncm-systemd/src/main/pan/components/systemd/schema.pan]: Can we do better than just string?

A quick check seems to indicate that this should be an uppercase string that starts with `CAP_`:
```
$ grep define /usr/include/linux/capability.h | sort
#define CAP_AUDIT_CONTROL    30
#define CAP_AUDIT_READ		37
#define CAP_AUDIT_WRITE      29
…
```

See also http://man7.org/linux/man-pages/man7/capabilities.7.html.

File: ncm-systemd/src/main/pan/components/systemd/schema.pan [hunk 2]
```diff
@@ -198,8 +198,10 @@ http://www.freedesktop.org/software/systemd/man/systemd.service.html
...
     'BusName' ? string
     'BusPolicy' ? string[] with length(SELF) == 2 && match(SELF[1], '^(see|talk|own)$')
+    'CapabilityBoundingSet' ? string[]
```
Comment (jrha) [inline ncm-systemd/src/main/pan/components/systemd/schema.pan]: Ditto.

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
@jrha you have some review comments left? maybe approve this PR and merge it once tests are completed?

**Pull request merged**